dask/dataframe/tests/test_shuffle.py

```diff
@@ -176,6 +176,37 @@ def test_partitioning_index_categorical_on_values():
     assert (res == res2).all()
 
 
+@pytest.mark.parametrize(
+    "npartitions", [1, 4, 7, pytest.param(23, marks=pytest.mark.slow)]
+)
+@pytest.mark.parametrize(
+    "column",
+    [
+        pytest.param(lambda df: "x", id="column name 'x'"),
+        pytest.param(lambda df: "y", id="column name 'y'"),
+        pytest.param(lambda df: df.x, id="series x"),
+        pytest.param(lambda df: df.y, id="series y"),
+        pytest.param(lambda df: df.x + df.y, id="series x + y"),
+        pytest.param(lambda df: df.x + 1, id="series x + 1"),
+        pytest.param(lambda df: df.index, id="df.index"),
+    ],
+)
+def test_set_index_general_paramed(npartitions, column, shuffle_method):
```

Comment:
This is a redundant (parametrized vs not) test that I created for my own benefit. I'll remove this as well.

Reply:
Oop, thanks for noticing this Paul, I forgot to go back through the PR to check for other extraneous changes. I've deleted this test and the breakpoint above.